### ISSUE TITLE
 Expensify chat - Incorrect LHN preview for Expensify chat

### ISSUE THREAD

**Issue by lanitochka17** [Expensify/App#43830]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 1.4.84-0
**Reproducible in staging?:** Y
**Reproducible in production?:** Y
**If this was caught during regression testing, add the test name, ID and link from TestRail:** N/A
**Issue reported by:** Applause - Internal Team

## Action Performed:
1. Go to staging.new.expensify.com
2. Go to Expensify chat

## Expected Result:
Expensify chat preview in LHN should display "Welcome! Let's get you set up." or "Message Concierge for help with setup"

## Actual Result:
Expensify chat preview in LHN displays "This is the beginning of your chat with Expensify"

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Native
- [ ] Android: mWeb Chrome
- [ ] iOS: Native
- [ ] iOS: mWeb Safari
- [ ] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

Add any screenshot/video evidence

</details> 

https://github.com/Expensify/App/assets/78819774/66151e8b-77a0-41d9-8233-54a68817a2de

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by lanitochka17** [comment]
We think that this bug might be related to #vip-vsp

**Comment by JmillsExpensify** [comment]
Opening up to the community. This is a #vip-vsb initiative.

**Comment by grgia** [comment]
@JmillsExpensify before I assign, what's the expected behavior here? 

It seems like no chats have been sent in the example video chat. Do we want to use the Welcome Text as the LHN preview? Techically "This is the beginning of your chat .." is correct for these

**Comment by JmillsExpensify** [comment]
Yeah, I think we are actually going to standardize on the welcome text very soon. Isn't that what you are working on @danielrvidal?

**Comment by grgia** [comment]
@danielrvidal bump on https://github.com/Expensify/App/issues/43830#issuecomment-2190511157

**Comment by grgia** [comment]
bumped in NewDot for confirmation

**Comment by danielrvidal** [comment]
I think I would still correct it with the text that is currently in there because that looks like a bug. We are planning to update the language but we don't have a formal plan yet so I would still fix it. I would think it would say "Welcome! Let's get you set up."

**Comment by grgia** [comment]
Great, going to assign this out

**Comment by JmillsExpensify** [comment]
Payment summary: 

Contributor: @suneox $250
Reviewer: @ZhenjaHorbach $250
